app/templates/components/explore-side-bar.hbs

```diff
@@ -0,0 +1,109 @@
+{{#ui-sticky}}
```

Comment:
No sticky for this please. When it's fully expanded sticky makes it difficult to scroll